public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/CompletionDataProvider.ts

```diff
@@ -4,7 +4,7 @@ import { HistoryItem } from '@grafana/data';
 import { escapeLabelValueInExactSelector } from 'app/plugins/datasource/prometheus/language_utils';
 
 import LanguageProvider from '../../../LanguageProvider';
-import { ExtractedLabelKeys, LokiQuery } from '../../../types';
+import { ParserAndLabelKeysResult, LokiQuery } from '../../../types';
```

Comment:
completions.ts has a few other usages that would benefit from this new type as well.

E.g.

```js
const { extractedLabelKeys, hasJSON, hasLogfmt, hasPack } = await dataProvider.getParserAndLabelKeys(query);
```